vscode-lean4/src/abbreviation/README.md

```diff
@@ -8,6 +8,11 @@ Use `$CURSOR` to set the new location of the cursor after replacement.
 If VS Code adds certain characters automatically (like `]` after typing `[`),
 ensure that each such subword is a strict prefix of another abbreviation.
 
+## Manual replacements
+
+When manually replacing an abbreviation early (using `Tab` by default), the abbreviation feature will pick the shortest abbreviation with a matching prefix. 
```

Comment:
I didn't know that `Tab` replaces an abbreviation! I use space, and regularly use space-backspace if I don't want the space.

Reply:
Early completions using space should apply the same logic, I believe.

Reply:
Yeah, that sounds plausible. Tab is really nice though.